pytorch_lightning/accelerators/accelerator.py

```diff
@@ -192,7 +200,9 @@ def setup_optimizers(self, model):
         if self.trainer.testing is True:
             return
 
-        optimizers, lr_schedulers, optimizer_frequencies = self.trainer.init_optimizers(model)
+        optimizers, lr_schedulers, optimizer_frequencies = self.trainer.init_optimizers(
+            model
```

Comment:
what is the reason for this change? have a single arg on new line...

Reply:
no need to block the PR for something small like this :)
We have 80+ others to look at

Reply:
@Borda these are all auto-formatting changes. I know it bloats the PR, but other attempts at explicitly splitting out the formatting changes from the core logic changes have also been clunky and invited questions as to whether we're black-formatting the whole codebase or not. I do strongly vouch for that, but I don't want to hold this up in the meantime. That's the main point of a standard formatter: so these PR review don't get bogged down by formatting